### PR TITLE
add easyblock for HEALPix

### PR DESCRIPTION
**Pull request by boegel**
required for https://github.com/hpcugent/easybuild-easyconfigs/pull/779

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyblocks/h/healpix.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            raise EasyBuildError("Failed to determine root for CFITSIO, module not loaded?")
+
+        self.comp_fam = self.toolchain.comp_family()
+        if self.comp_fam == toolchain.INTELCOMP:  #@UndefinedVariable
+            cxx_config = '4'  # linux_icc
+
+        elif self.comp_fam == toolchain.GCC:  #@UndefinedVariable
+            cxx_config = '2'  # generic_gcc
+
+        else:
```
Comment (wpoely86) [inline easybuild/easyblocks/h/healpix.py]: less whitespace in this if-else block?

File: easybuild/easyblocks/h/healpix.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            r"Available configurations for C\+\+ compilation are:[\s\n\S]*Choose one number:": cxx_config,
+            r"PGPLOT.[\s\n]*Do you want to enable this option \?[\s\n]*\([^)]*\) \(y\|N\)": 'N',
+            r"the parallel implementation[\s\n]*Enter choice.*": '1',
+        }
+        run_cmd_qa(cmd, qa, std_qa=std_qa, log_all=True, simple=True, log_ok=True)
+
+    def build_step(self):
+        """Custom build procedure for HEALPix."""
+        # disable parallel build
+        self.cfg['parallel'] = '1'
```
Comment (wpoely86) [inline easybuild/easyblocks/h/healpix.py]: print this?

**Comment by boegel** [comment]
tested with https://github.com/hpcugent/easybuild-easyconfigs/pull/779, good to go, thanks for the review @wpoely86!

**Pull request merged**